functions/Get-DbaDatabase.ps1

```diff
@@ -342,8 +358,10 @@ function Get-DbaDatabase {
 
                     $backupStatus = $null
                     if ($NoFullBackup -or $NoFullBackupSince) {
-                        if ($db.Name -cin $lastCopyOnlyBackups.Database) {
-                            $backupStatus = "Only CopyOnly backups"
+                        foreach ($dif in $lastCopyOnlyBackups.Database) {
+                            if ($server.getStringComparer($server.Collation).Compare($db.Name, $dif) -eq 0 ) {
+                                $backupStatus = "Only CopyOnly backups"
+                            }
                         }
                     }
```

Comment:
This code Never does anything,  any database without a regular  Full Backups were  filtered out of $inputObject already Line 325